app/controllers/v1/resources_controller.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'base64'
+require 'digest'
```

Comment:
Where is this used?

Reply:
@jcoyne good catch, cruft. This was from early debugging. Removed. Thanks for the heads up.